Dockerfile

```diff
@@ -21,6 +21,10 @@ RUN apt-get update -qqy && \
     && apt-get autoremove \
     && rm -rf /var/lib/apt/lists/*
 
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
I have tested; it failed. Please add `curl` at line 14, @jeremi.
And you should update commit messages and PR title following [Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

Reply:
@jeremi , @ducminhle, can we directly install the `cargo` by using the apt command.
```
apt-get install -y cargo
```